From 0bc5f5bfc7ea39dff9d6e281b70a5902d77fa19d Mon Sep 17 00:00:00 2001
From: Mystro256 <alexjnewt@hotmail.com>
Date: Tue, 25 Jan 2022 16:32:19 -0500
Subject: [PATCH] Link libclang dynamically

Instead of linking statically to clangFrontendTool, since Fedora doesn't
provide static clang libraries.
---
 lib/comgr/CMakeLists.txt | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/lib/comgr/CMakeLists.txt b/lib/comgr/CMakeLists.txt
index f31bdeb..abdb292 100644
--- a/lib/comgr/CMakeLists.txt
+++ b/lib/comgr/CMakeLists.txt
@@ -277,7 +277,8 @@ install(FILES
   DESTINATION "${AMD_COMGR_PACKAGE_PREFIX}")
 
 set(CLANG_LIBS
-  clangFrontendTool)
+  clang
+  clang-cpp)
 
 set(LLD_LIBS
   lldELF
-- 
2.31.1

